src/System.CommandLine/ParseResultExtensions.cs

```diff
@@ -9,6 +9,36 @@ namespace System.CommandLine
 {
     public static class ParseResultExtensions
     {
+        public static object GetDefaultValue(this Type type)
```

Comment:
This would overload the concept of "default value", which currently means the default for the argument, not for the type.

Reply:
Understand the naming concern.  We could:

- rename it
- make it private where I need it 

I prefer the first because if someone needs this behavior (and we're tossing a lot of objects around with the reflection work) I'd like it done this way.